sklearnex/linear_model/ridge.py

```diff
@@ -15,3 +15,10 @@
 # ===============================================================================
 
 from daal4py.sklearn.linear_model import Ridge
+from onedal._device_offload import support_usm_ndarray
+
+# TODO:
+# implement GPU offloading via onedal4py backend.
+Ridge.fit = support_usm_ndarray(queue_param=False)(Ridge.fit)
+Ridge.predict = support_usm_ndarray(queue_param=False)(Ridge.predict)

```

Comment:
General comment for several files - why is it necessary to decorate these functions?

Reply:
For dpnp/dpctl, aka usm_ndarrays interop.